src/api/customer/customer-controller.ts

```diff
@@ -15,6 +15,39 @@ const createCustomer = (): CustomerCreationRequestHandler[] => {
     ]
 }
 
+const findAllCustomers = (): CustomersFindAllRequestHandler[] => {
+    return [
+        findAllCustomerValidator(),
+        persistFindAllCustomers,
+        findAllCustomerSerializer,
+        (req, res) => { 
+            if(res.locals.customerToRespond == undefined){
+                res.status(NOT_FOUND).json(res.locals.customerError)
+            } else {
+                res.status(OK).json(res.locals.customerToRespond)
+            }
```

Comment:
Poderia parar de testar com `undefined` e reverter a ordem, para não precisar negar o `if`.
```suggestion
            if (res.locals.customerToRespond) {
                res.status(OK).json(res.locals.customerToRespond)
            } else {
                res.status(NOT_FOUND).json(res.locals.customerError)
            }
```